### ISSUE TITLE
Complete the assignment 2

### ISSUE THREAD

**Issue by nttan251290** [nttan251290/ios_yelp_swift-master#1]
Hi Charles ,
I've finished the assignment 2 . The core requirement  is finished. But the optional I don't have enough time , I will try to research more that :( . /cc @chug2k

Thank you.

**Comment by chug2k** [comment]
I cannot open your assignment: `Interface Builder could not open the document Main.storyboard" because it does not exist.`

Can you please fix?

**Comment by nttan251290** [comment]
Yeah , it's strange . I tested it again when commit some day ago. I will check it tonight. I'm sorry about this inconvenient..
